Cargo.toml

```diff
@@ -50,6 +50,7 @@ indoc = "2.0"
 js-sys = "0.3"
 libfuzzer-sys = "0.4"
 log = "0.4"
+itertools = "0.10.5"
```

Comment:
The dependency on itertools can be removed if you use the `entry().or_default()` pattern in the capabilities check.

Reply:
`and_modify` and `or_insert` can be used without itertools. I included it to be able to sort the generated errors to get an expected order because tests that yield multiple errors are flaky if the errors are not in an expected order.

Reply:
itertools dependency removed.